### DISCUSSION TITLE
reactions plugin error

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#596] [Q&A] [answered]
I'm consistently seeing this plugin only show an error on https://github.com/jayvdb/jayvdb. I've reduced the settings down to
```yaml
plugin_reactions_limit: 50
plugin_reactions_limit_issues: 25
```

**Comment by lowlighter** [comment, marked as answer]
Yeah this is still related to #584 

Currently only the `base` plugin has been patched to attempt to avoid GitHub API timeout (so at least Metrics can be partially used).
Other plugins will be tracked in #598 👍
